app/uk/gov/hmrc/nisp/views/main.scala.html

```diff
@@ -65,16 +66,19 @@
 }
 
 @betaBanner = {
-    @if(showBetaBanner) {
-        @layouts.betaBanner(userLoggedIn,
-            applicationConfig.betaFeedbackUrl,
-            applicationConfig.betaFeedbackUnauthenticatedUrl,
-            true)
+    @if(user.isDefined) {
+        @uk.gov.hmrc.nisp.config.wiring.NispCachedStaticHtmlPartialRetriever.getPartialContent(ApplicationConfig.generateHeaderUrl()(request, user.get))
```

Comment:
`NispCachedStaticHtmlPartialRetriever` should be abstracted out, so it can be overridden in the tests